Relax package.json grunt peerDependencies

```
$ npm install grunt-jscs-checker --save-dev
npm ERR! peerinvalid The package grunt does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-contrib-concat@0.3.0 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-uglify@0.2.2 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-contrib-jshint@0.6.0 wants grunt@~0.4.0
npm ERR! peerinvalid Peer grunt-jscs-checker@0.3.2 wants grunt@0.4.2
npm ERR! node -v v0.10.13
npm ERR! npm -v 1.3.2
npm ERR! code EPEERINVALID
npm ERR! not ok code 0
```

From [the blog post on peer dependencies](http://blog.nodejs.org/2013/02/07/peer-dependencies/):

> The best way to determine what your peer dependency requirements should be is to actually follow semver. Assume that only changes in the host package's major version will break your plugin. Thus, if you've worked with every 1.x version of the host package, use "~1.0" or "1.x" to express this. If you depend on features introduced in 1.5.2, use ">= 1.5.2 < 2".

So, package.json line 37 should be something like `"grunt": "~0.4.0"` or `"grunt": ">= 0.4.2 < 0.5"`

EDIT: (It's worth pointing out that upgrading the packages reported above resolves this problem, at least at the time of writing. Still, it's nice to future-proof.)

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
         "load-grunt-tasks": "0.2.0"
     },
     "peerDependencies": {
-        "grunt": "0.4.2"
+        "grunt": "~0.4.2"
     },
     "main": "Gruntfile.js",
     "keywords": [

